apps/pie-docs/component-statuses.json

```diff
@@ -0,0 +1,24 @@
+{
```

Comment:
This file can be removed 🎉.

A PR was recently introduced to change how component statuses are fetched https://github.com/justeattakeaway/pie/pull/2034 :)

Reply:
deleted